### ISSUE TITLE
No birthday sync to IOS

### ISSUE THREAD

**Issue by Concific** [nextcloud/contacts#850]
**Describe the bug**

I cannot sync the birthday informations in contacts to IOS device (iPhone IOS 12.1.2)
After synchronize the contacts to IOS, the birthday fields on my mobile are empty. All other informations (name, number etc.) will work properly.

**To Reproduce**

1. Go to any contact or add a new contact in contacts-app
2. Add a birthday date in birthday field.
3. Sync to IOS device
4. Birthday informations in contacts on mobile device are missing

**Server configuration**

Operating system: Linux
Web server: Apache
Database: MySQL
PHP version: 7.2
Nextcloud version: 15.0.0
Contacts version: 3.0.1
Updated from an older Nextcloud or fresh install: fresh install

**Comment by skjnldsv** [comment]
Hello! Any news here?
Is this still happening?

**Comment by JC5** [comment]
This is also happening to me. There's not much more to add I think except it still happens in the latest release.

**Comment by JC5** [comment]
This is the card generated by Nextcloud when I press download:

```
BEGIN:VCARD
VERSION:3.0
PRODID:-//Sabre//Sabre VObject 4.1.6//EN
UID:e9e0cc90-6926-4d07-b3a9-674383cab715
FN:James Cole
N:Cole;James;;;
BDAY:20160126T000007
END:VCARD
```

Check out the time added to the birthday.

**Comment by JC5** [comment]
Manually editing the card to `BDAY:20160126` in the database fixes this issue. My uninformed guess is that the time selector on the birthday field is messing things up.

**Comment by JC5** [comment]
I also ended up with this card somehow:

```
BEGIN:VCARD
VERSION:3.0
FN:112
N:;112;;;
TEL;TYPE="WORK,VOICE":112
ITEM1.ORG:Alarmcentrale
NOTE:112 Alarmcentrale
UID:41893e41-0cb1-4877-af93-645f07800e5c
REV:20181117T133714Z
CATEGORIES:Bedrijven
BDAY;VALUE=DATE-AND-OR-TIME:2019-02-02T21:10:31
END:VCARD
```

And yes I have a card for the emergency number. 😋

**Comment by JC5** [comment]
See also issue #813 for similar problems.

**Comment by Concific** [comment]
Still the same after update! No birthday entries in IOS.

**Comment by beinaendi** [comment]
iOS can only show birthdays without time. In v-card or with synching time is transmitted unless it is 00:00. So you just have to manually take away any different time than 00:00. Had the same issue for a long time but the perfect solution would of course be if it was corrected on nextcloud web service.

**Comment by Concific** [comment]
I tried it and It doesn´t matter if the time is 00:00 or different. No sync to IOS.

**Comment by beinaendi** [comment]
In my case it does matter and by the way the birthdays are always synced. I can see birthdays in iPhone's birthday calendar always, but they are only displayed in contact details when they are with time 00:00. Can you see them in your birthday calendar if you activate it?

**Comment by skjnldsv** [comment]
Should be fixed with 9057462c34977103a2c68124ee8f50a2d1a967ce
